files with spaces in names cause `pr-comment.yaml` workflow to fail

This [workflow from managing-computational-projects](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754) has errored in a fun way:

```
cd built/
  git config --local user.email "actions@github.com"
  git config --local user.name "GitHub Actions"
  CURR_HEAD=$(git rev-parse HEAD)
  git checkout --orphan md-outputs-PR-${NR}
  git add -A 
  git commit -m "source commit: ${CURR_HEAD}"
  ls -A | grep -v '^.git$' | xargs rm -r
  cd ..
  unzip -o -d built built.zip
  cd built
  git add -A
  git commit --allow-empty -m "differences for PR #${NR}"
  git push -u --force --set-upstream origin md-outputs-PR-${NR}
  shell: /usr/bin/bash -e {0}
  env:
    NR: 89
Switched to a new branch 'md-outputs-PR-89'
[md-outputs-PR-89 (root-commit) a[3](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:3)fe9d0] source commit: e7690a50e06a5b12eaeb553c900c1b2723eb320[4](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:4)
 66 files changed, 3[5](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:5)[6](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:6)5 insertions(+)
 create mode 100644 01-introduction.md
 create mode 100644 02-motivation.md
 create mode 100644 03-overview.md
 create mode 100644 04-FAIRHowTo.md
 create mode 100644 04-reproducibility.md
 create mode 100644 05-versioncontrol.md
 create mode 100644 0[7](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:7)-setup.md
 create mode 100644 0[8](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:8)-KanbanTutorial.md
 create mode 100644 08-collaborativeworking.md
 create mode 100644 0[9](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:9)-GithubCodeReview.md
 create mode [10](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:10)0644 09-rdm.md
 create mode 100644 10-documentation.md
 create mode 100644 [11](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:11)-rigour.md
 create mode 100644 [12](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:12)-analysis.md
 create mode 100644 [13](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:13)-codereview.md
 create mode 100644 [14](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:14)-codemodularity.md
 create mode 100644 [15](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:15)-publication.md
 create mode 100644 [16](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:16)-open.md
 create mode 100644 [17](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:17)-citation.md
 create mode 100644 51-datawrangling.md
 create mode 100644 CODE_OF_CONDUCT.md
 create mode 100644 LICENSE.md
 create mode 100644 config.yaml
 create mode 100644 discuss.md
 create mode 100644 fig/IMP-CodeReviewAsynch.png
 create mode 100644 fig/IMP-CodeReviewGroup.png
 create mode 100644 fig/IMP-CodeReviewPair.png
 create mode 100644 fig/IMP-codefastorwell.png
 create mode 100644 fig/IMP-kanbanResearch.png
 create mode 100644 fig/IMP-pipeline1.png
 create mode 100644 fig/IMP-pipeline2.png
 create mode 100644 fig/ReproducibleJourney.jpg
 create mode 100644 fig/ai-for-research.jpg
 create mode 100644 fig/analysis-workflow.jpg
 create mode 100644 fig/astropy-ecosystem.jpg
 create mode 100644 fig/autspaces-research-workflow.jpg
 create mode 100644 fig/citizen-science.jpg
 create mode 100644 fig/data-manager.jpg
 create mode 100644 fig/data-mining.jpg
 create mode 100644 fig/data-safe-haven.jpg
 create mode 100644 fig/datasciencespecifics.jpg
 create mode 100644 fig/detecting-harmful-contents.jpg
 create mode 100644 fig/digital-twin.jpg
 create mode 100644 fig/documentation.jpg
 create mode 100644 fig/global-collaboration.jpg
 create mode 100644 fig/handbook-for-application.jpg
 create mode 100644 fig/healthcare-data.jpg
 create mode 100644 fig/healthy-research-tree.jpg
 create mode 100644 fig/machine-learning.jpg
 create mode 100644 fig/participatory-research.jpg
 create mode 100644 fig/reproducible-matrix.jpg
 create mode 100644 fig/research-cycle.jpg
 create mode 100644 fig/research-evaluation.jpg
 create mode 100644 fig/research-foundation.jpg
 create mode 100644 fig/research-outputs.jpg
 create mode 100644 fig/skill-spectrum.jpg
 create mode 100644 fig/ttw-welcome.jpg
 create mode 100644 fig/utility-vs-privacy.jpg
 create mode 100644 files/2[20](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:21)201_TrainingMaterials.md
 create mode 100644 index.md
 create mode 100644 instructor-notes.md
 create mode 100644 learner-profiles.md
 create mode 100644 md5sum.txt
 create mode 100644 notes for issue
 create mode 100644 reference.md
 create mode 100644 setup.md
rm: cannot remove 'notes': No such file or directory
rm: cannot remove 'for': No such file or directory
rm: cannot remove 'issue': No such file or directory
Error: Process completed with exit code 1[23](https://github.com/carpentries-incubator/managing-computational-projects/actions/runs/4222418855/jobs/7330902754#step:5:24).
```

The culprit is a file called [`episodes/notes for issue`](https://github.com/carpentries-incubator/managing-computational-projects/blob/7e30c77137de461de86dbd225c3b7f8c9ec282f9/episodes/notes%20for%20issue)

While filenames with spaces are generally a nono, it's not {sandpaper}'s job to judge. This should be fixed by using `xargs -I _ rm -r '_'` to make sure to quote the spaces.

--- a/DESCRIPTION
+++ b/DESCRIPTION
@@ -1,6 +1,6 @@
 Package: sandpaper
 Title: Create and Curate Carpentries Lessons
-Version: 0.11.8
+Version: 0.11.9
 Authors@R: c(
     person(given = "Zhian N.",
            family = "Kamvar",

--- a/NEWS.md
+++ b/NEWS.md
@@ -1,3 +1,12 @@
+# sandpaper 0.11.9 (in development)
+
+CONTINUOUS INTEGRATION
+----------------------
+
+* Lessons with files that have spaces in their names (e.g as a learning tool)
+  no longer fail to comment on pull request previews (reported: @zkamvar, #399;
+  fixed: @zkamvar, #400)
+
 # sandpaper 0.11.8 (2023-03-09)
 
 ## BUG FIX

--- a/inst/workflows/pr-comment.yaml
+++ b/inst/workflows/pr-comment.yaml
@@ -21,8 +21,8 @@ jobs:
   test-pr:
     name: "Test if pull request is valid"
     runs-on: ubuntu-latest
-    if: > 
-      github.event.workflow_run.event == 'pull_request' && 
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
       github.event.workflow_run.conclusion == 'success'
     outputs:
       is_valid: ${{ steps.check-pr.outputs.VALID }}
@@ -104,9 +104,9 @@ jobs:
           git config --local user.name "GitHub Actions"
           CURR_HEAD=$(git rev-parse HEAD)
           git checkout --orphan md-outputs-PR-${NR}
-          git add -A 
+          git add -A
           git commit -m "source commit: ${CURR_HEAD}"
-          ls -A | grep -v '^.git$' | xargs rm -r
+          ls -A | grep -v '^.git$' | xargs -I _ rm -r '_'
           cd ..
           unzip -o -d built built.zip
           cd built
@@ -129,7 +129,7 @@ jobs:
         with:
           run: ${{ github.event.workflow_run.id }}
           name: 'diff'
-          
+
       - if: ${{ steps.dl.outputs.success == 'true' }}
         run: unzip ${{ github.workspace }}/diff.zip
 
@@ -138,7 +138,7 @@ jobs:
         if: ${{ steps.dl.outputs.success == 'true' }}
         uses: carpentries/actions/comment-diff@main
         with:
-          pr: ${{ env.NR }} 
+          pr: ${{ env.NR }}
           path: ${{ github.workspace }}/diff.md
 
   # Comment if the PR is open and matches the SHA, but the workflow files have

